src/main/java/com/tictactoe/UserInputValidator.java

```diff
@@ -18,8 +18,4 @@ public boolean playersHaveUniqueSymbols(String player1Symbol, String player2Symb
     public boolean validateCellSelection(int cellSelectionInput) {
         return 0 < cellSelectionInput && cellSelectionInput < 10 && (this.board.getSymbol(cellSelectionInput-1).equals(" "));
     }
-
-    public boolean validateGameTypeSelection(int gameTypeInput) {
-        return 1 <= gameTypeInput && gameTypeInput <= 4;
-    }
```

Comment:
I never used it. I think I was planning to, but I went a different direction and just completely forgot that I had this method here.